src/TouristInfoCards.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import TouristicInfoCard from "./TouristicInfoCard";
+
+const TouristInfoCards = () => (
+  <div className="card-container">
+    <TouristicInfoCard
+      src="https://images.unsplash.com/photo-1531152369337-1d0b0b9ef20d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80"
+      link="https://peoplemakeglasgow.com/"
+      cityName="Glasgow"
+      alt="Glasgow Image"
+    />
+    <TouristicInfoCard
+      src="https://images.unsplash.com/photo-1515586838455-8f8f940d6853?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1884&q=80"
+      link="https://www.visitmanchester.com/"
+      cityName="Manchester"
+      alt="Manchester Image"
+    />
+    <TouristicInfoCard
+      src="https://images.unsplash.com/photo-1513635269975-59663e0ac1ad?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
+      link="https://www.visitlondon.com/"
+      cityName="London"
+      alt="London Image"
+    />
+  </div>
+);
```

Comment:
Good job. I like the way you created this component and is reusing it. What if we created a data structure which we could map and return this component? Wouldn't be cool? We could avoid copy and paste this component over and over again.

Something like:
```js
const touristInformations = [{
   city: 'Glasgow',
   site: 'https://peoplemakeglasgow.com/',
   imageURL: '"https://frontline-collections.com/wp-content/uploads/2018/02/Glasgow-Debt-Collection.jpg'
}, {
   city: 'Manchester',
   site: 'https://www.visitmanchester.com/',
   imageURL: 'https://www.prolificnorth.co.uk/sites/default/files/styles/lightbox_large/public/images/news/manchester-cityscape0.jpg?itok=VjQDdTYJ'
}];
```

What do you think?